core/src/components/range/range-interface.ts

```diff
@@ -8,7 +8,15 @@ export interface RangeChangeEventDetail {
   value: RangeValue;
 }
 
+export interface RangeChangeStartEventDetail {
```

Comment:
~ Thoughts on re-using the existing `RangeChangeEventDetail`? Unsure of the value of separate interfaces, if the type signature is identical.

Reply:
We'd agreed to use separate event details on the design doc, to avoid potential confusion and make them easier to update if we need new fields later. I'm open to changing it though if y'all don't think it'll look weird down the road 🤷‍♀️